pkg/manifests/manifests.go

```diff
@@ -42,7 +40,7 @@ const (
 	// OwningClusterIngressLabel should be applied to any objects "owned by" a
 	// clusteringress to aid in selection (especially in cases where an ownerref
 	// can't be established due to namespace boundaries).
-	OwningClusterIngressLabel = "ingress.openshift.io/clusteringress"
+	OwningClusterIngressLabel = "ingress.operator.openshift.io/clusteringress"
```

Comment:
if we're changing this, what about also changing `/clusteringress` to `/ingresscontroller` or something?